src/memray/_memray/sink.cpp

```diff
@@ -167,7 +170,7 @@ std::unique_ptr<Sink>
 FileSink::cloneInChildProcess()
 {
     std::string file_name = d_fileNameStem + "." + std::to_string(::getpid());
-    return std::make_unique<FileSink>(file_name, true);
+    return std::make_unique<FileSink>(file_name, true, d_compress);
```

Comment:
I'm wondering if we might want to hardcode `false` instead of passing `d_compress` here. We might get killed in the middle of compressing the file if we're a `multiprocessing` worker, and that'll leave both the compressed file and the uncompressed file on disk if so. Ignoring the setting has the better failure mode...

Reply:
But compression only will trigger if the tracker is destroyed which either happens at the end of the lifetime of the child or if there is a context manager somewhere. I cannot imagine a situation where we will get killed mid compression in a natural way... What do you think?

Reply:
That's a fair point - the normal case for a `multiprocessing` worker will be that the Tracker destructor doesn't fire, because we never return to the frames in the parent where the context manager was entered, and so we _never_ try to compress.

Fair point.